codice/alliance
# CAL-249 Update the video streaming code to configure chunks in megabytes

#### What does this PR do?
Update the video streaming code to configure chunks in megabytes instead of bytes.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@bdeining 
@jrnorth 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@lessarderic
#### How should this be tested?
1) Build and install.
2) With both the FMV Admin Plugin and the video stream configuration page, create a stream monitor. Note that the default values are now in megabytes. Re-open the config and confirm that the re-loaded values for the chunk size are correct. 
3) Stream video and confirm that the chunk sizes are as expected.

#### Any background context you want to provide?
#### What are the relevant tickets?

[CAL-249](https://codice.atlassian.net/browse/CAL-249)
#### Screenshots (if appropriate)
#### Checklist:
